frontend/src/metabase/core/components/Radio/index.ts

```diff
@@ -1 +1,2 @@
 export { default } from "./Radio";
+export { RadioGroup } from "./Radio.styled";
```

Comment:
You don't need to do that. Because `Radio` accepts `className` you can style it directly.